Tavisca.Bootcamp.LanguageBasics.Exercise3/Program.cs

```diff
@@ -1,10 +1,25 @@
 ﻿using System;
 using System.Linq;
-
+using System.Collections.Generic;
 namespace Tavisca.Bootcamp.LanguageBasics.Exercise1
 {
+    
     public static class Program
     {
+        public class Food{
+            public int pro;
+            public int carb;
+            public int fat;
+            public int cal;
+            public int ind;
+            public Food(int p,int c,int f,int i){
+                pro=p;
+                carb=c;
+                fat=f;
+                ind=i;
+                cal=(f*9)+((p+c)*5);

```

Comment:
You should not write business logic in constructor...Move it to a method, say 'GetCalories(protein, carb, fat, menuIndex) and return calculated calories.